server/routes/createPlan/abilityToWork/index.ts

```diff
@@ -4,23 +4,21 @@ import parseCheckBoxValue from '../../../middleware/parseCheckBoxValue'
 import getPrisonerByIdResolver from '../../../middleware/resolvers/getPrisonerByIdResolver'
 import type { Services } from '../../../services'
 import AbilityToWorkController from './abilityToWorkController'
-import getCiagPlanByIdResolver from '../../../middleware/resolvers/getCiagPlanByIdResolver'
 import checkSessionPageData from '../../../middleware/checkSessionPageData'
 
 export default (router: Router, services: Services) => {
   const controller = new AbilityToWorkController(services.ciagService)
 
   router.get(
     '/plan/create/:id/ability-to-work/:mode',
-    [getPrisonerByIdResolver(services.prisonerSearchService), getCiagPlanByIdResolver(services.ciagService)],
+    [getPrisonerByIdResolver(services.prisonerSearchService)],
```

Comment:
This is one half of the fix - basically every route has it's own route definition file, and for the `GET` and `POST` requests it uses the resolver (sic!) to retrieve the CIAG Plan (Induction)

Because there is no differentiation here between the create journey and the update journey, this is why currently we have the problem where it tries to retrieve the Induction on every page load, regardless of journey.

What I've done here is go through all the individual route definition files, and remove the call to the resolver